docs/guides/XcmTransfers.md

```diff
@@ -0,0 +1,61 @@
+# 💸 Cross Chain Transfers with xTokens
```

Comment:
How does a user arrive at the XcmTransfers page? Please add to sidebar

Reply:
Indeed, adding.